jpa2ddl-core/src/main/java/com/devskiller/jpa2ddl/GenerationMode.java

```diff
@@ -5,7 +5,12 @@
 	/**
 	 * Generation based on setting up embedded database and dumping the schema
 	 */
-	DATABASE,
+	EMBEDDED_DATABASE,
```

Comment:
Maybe an overkill, but this isn't backward compatible? This `enum` is used as part of configuration?

Reply:
It is, but as we're releasing the stable version I don't have problem with breaking it